### PR TITLE
G-Mode: Big Pink part 2

### PR DESCRIPTION
**Pull request by kjbranch**
Added strats: 
- to 4, 10
- from 6
- break B

Still need:
- to 8, 12
- from 8, 9
- remote acquire 10, 11

### CODE REVIEW

File: region/brinstar/pink.json [hunk 1]
```diff
@@ -2133,6 +2225,189 @@
...
+                    "Overload PLMs by moving through the camera scroll blocks in front of the passageway leading to the exit of Spore Spawn Supers.",
+                    "Go to the left end of the hallway outside of the door to the Pink Hopper Room, then use Spring Ball (without HiJump) to jump partially through the crumble blocks,",
+                    "and place bombs at the top of the jump to safely kill the Sidehoppers. Then IBJ up to the item above."
+                  ],
+                  "devNote": "An equivalent strat with using a Power Bomb isn't needed as it is the same requirements as getting to 4 with the blocks broken."
+                },
+                {
+                  "name": "G-mode Morph with Ceiling Bomb Jump",
+                  "notable": false,
+                  "requires": [ "canCeilingBombJump" ],
```
Comment (osse101) [inline region/brinstar/pink.json:2369]: `h_canArtificialMorphCeilingBombJump`
also
`h_canArtificialMorphDiagonalBombJump`

File: region/brinstar/pink.json [hunk 2]
```diff
@@ -2133,6 +2225,189 @@
...
+                  "note": "Ceiling bomb jump across the chasm to avoid the Sidehoppers."
+                },
+                {
+                  "name": "G-mode Morph with Ceiling Bomb Jump, Break Power Bomb Blocks",
+                  "notable": false,
+                  "requires": [
+                    "canCeilingBombJump",
```
Comment (osse101) [inline region/brinstar/pink.json:2376]: here too

File: region/brinstar/pink.json [hunk 3]
```diff
@@ -2133,6 +2225,189 @@
...
+                  ]
+                },
+                {
+                  "name": "G-mode Morph with IBJ and Power Bombs",
+                  "notable": false,
+                  "requires": [
+                    "h_canArtificialMorphIBJ",
+                    {"ammo": {"type": "PowerBomb", "count": 1}}
+                  ],
+                  "note": "Use a Power Bomb to kill one or both Sidehoppers in the pit below, roll off and kill the second with Bombs, then IBJ up to the item above.",
```
Comment (osse101) [inline region/brinstar/pink.json:2392]: Killing one or both with Bombs should be the same difficulty

File: region/brinstar/pink.json [hunk 4]
```diff
@@ -2133,6 +2225,189 @@
...
+              "id": 13,
+              "strats": [
+                {
+                  "name": "G-mode Morph Power Bomb the Blocks",
+                  "notable": false,
+                  "requires": [ {"ammo": {"type": "PowerBomb", "count": 2}} ],
+                  "clearsObstacles": ["B"],
+                  "note": [
+                    "Use a Power Bomb to kill any Sidehoppers in the pit below, then roll off and use a second, while exiting G-Mode to destroy the blocks in front of the door.",
+                    "Place the Power Bomb far enough left to hit the blocks, but not so far as to lure the Sidehopper. The wall to the right should just barely or paritally be on camera."
```
Comment (osse101) [inline region/brinstar/pink.json:2407]: Squiggle plant on the right is the usual cue.  But that is gone with Mosaic 🤔

**Review by osse101 — COMMENTED** [review]
Mainly, requirements can be reduced on many of the strats.  The two hoppers aren't that bad with bombs, or rolling them off camera.  Compared to the two at the top which are much harder to get past.  Killing the hoppers from below with Springball is easier but maybe excessive.

2->15 can overload PLMs using a bombHorizontally with the tunnel at 1.  Then you have 1->15 and 2->15 with PLMs already overloaded.

**Comment by kjbranch** [comment]
When I was first practicing this, the hoppers kept following me back to the right after luring them to the left. It looks like the camera can be a bit finicky in here.

**Pull request merged**